packages/core/android/src/main/kotlin/com/datadog/reactnative/DatadogSDKWrapper.kt

```diff
@@ -54,19 +98,27 @@ internal class DatadogSDKWrapper : DatadogWrapper {
         configuration: Configuration,
         consent: TrackingConsent
     ) {
-        Datadog.initialize(context, configuration, consent)
+        val core = Datadog.initialize(context, configuration, consent)
+        DatadogSDKWrapperStorage.setSdkCore(core as InternalSdkCore)
+        DatadogSDKWrapperStorage.notifyOnInitializedListeners()
     }
 
     override fun enableRum(configuration: RumConfiguration) {
-        Rum.enable(configuration)
+        DatadogSDKWrapperStorage.getSdkCore()?.let {
```

Comment:
`getSdkCore` already returns non-null value, there is no need for `.?` here. Same applies for the similar calls below?

```suggestion
        DatadogSDKWrapperStorage.getSdkCore().let {
```